corehq/ex-submodules/casexml/apps/phone/xml.py

```diff
@@ -135,7 +135,8 @@ def get_registration_element_for_case(case):
 
 def get_data_element(name, dict):
     elem = safe_element(name)
-    for k, v in dict.items():
+    # sorted for deterministic unit tests
+    for k, v in sorted(dict.items()):
```

Comment:
I think this is a bad place to add sorting. Sorting is a relatively expensive operation (`O(n log n)`). It can be added to tests when absolutely necessary to support Python 2 and 3.

Reply:
I was hoping to get to this, but it looks unlikely now.